packages/node_modules/nav-frontend-skjema/src/textarea-controlled.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import * as PT from 'prop-types';
+import { omit } from 'nav-frontend-js-utils';
+import Textarea from './textarea';
+import {SkjemaelementFeil} from "nav-frontend-skjema/src/skjemaelement-feilmelding";
+
+
+export interface TextareaControlledProps {
```

Comment:
Burde arve fra `HTMLAttributes<HTMLTextareaElement>` eller tilsvarende?

Reply:
Ja, denne var litt spesiell. Tenkte at man kunne arver `TextareaProps`, men når man ser på `proptypes` lengre ned så ser man at `TextareaControlled` ikke vil ha inn `value` og `onChange`. Prøvde meg på en løsning hvor man bare delvis arver (https://stackoverflow.com/questions/48215950/exclude-property-from-type), men det funket ikke så bra da det var to keys man ikke ville arve.

Enig at den hvertfall burde arve fra `HTMLAttributes<HTMLTextareaElement>`.